runtime/pointers.ts

```diff
@@ -587,7 +587,13 @@ export class PointerProperty<T=any> extends Ref<T> {
     // get current pointer property
     public override get val():T {
         // this.handleBeforePrimitiveValueGet();
-        return this.pointer.getProperty(this.key, this.#leak_js_properties);
+        const val = this.pointer.getProperty(this.key, this.#leak_js_properties);
+        if (this.pointer instanceof LazyPointer) return "lazy..."
```

Comment:
Missing logic? Is this a problem?

Reply:
Fixed in https://github.com/unyt-org/datex-core-js-legacy/pull/61